clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupOptions.java

```diff
@@ -45,10 +46,21 @@ public RemoveMembersFromConsumerGroupOptions() {
         this.members = Collections.emptySet();
     }
 
+    /**
+     * Sets an optional reason.
+     */
+    public void reason(final String reason) {
+        this.reason = reason;
```

Comment:
We have a `reason` setter and getter in the class. I think we should name them differently for other developer better distinguishing between them, instead of overloading them with different purpose. how about `setReason` and `reason`?

Reply:
We have to follow the KIP on the naming here. We usually don't use "set" here.

Reply:
yeah, i was on the fence for this. reverting

Reply:
OK, if it's following the KIP,  then I'm fine with it. Thanks.